### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by msamosij**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# [tedee Release Notes](https://tedee-com.github.io/tedee-release-notes/)
```
Comment (konradsikorski) [inline README.md:1]: Add an introduction section where you describe what is this repo about. It contains release history for Tedee Android and iOS app, firmware changes for Tedee devices. The release notes are available here.

Comment (msamosij) [reply README.md:1]: done

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
```
Comment (konradsikorski) [inline README.md:1]: Either remove this line (and change tedee to Tedee) or remove line number 3

Comment (msamosij) [reply README.md]: done

**Review by konradsikorski — APPROVED** [review]

**Pull request merged**